### ISSUE TITLE
No dialog for blocks that need configuration

### ISSUE THREAD

**Issue by herbdool** [backdrop-contrib/blockreference#4]
I tested with adding *Existing Content* block. Got errors like `Trying to get property 'title' of non-object in NodeBlock->getTitle() (line 38 of /app/core/modules/node/node.block.inc).` then I realized that I wasn't given the option to select the node. I'm not sure what is the best approach here. Did it provide a dialog in D7? Or just put a warning message?

**Comment by ghost** [comment]
Yeah, I tried to use this on a block that requires configuration, and it's just not usable without this option...

**Comment by yorkshire-pudding** [comment]
+1 on Existing Content

**Comment by yorkshire-pudding** [comment]
Now that entity reference is in core and can do a similar task, I think that using a block reference field to display content via the existing content block may be an overly circuitous route.  However, there ~~will~~ _may_ be other blocks that need some basic config that the module currently can't do so worth keeping open.  It would be good to document any blocks that people want to use but can't.
